portfolio.css

```diff
@@ -0,0 +1,264 @@
+* {
+	box-sizing: border-box;
+	font-family: 'Nanum Gothic',sans-serif;
+}
+a {
+	text-decoration: none;
+	color: inherit;
+}
+.sticky {
+	width: 100%;
+	background-color:#000000;
+}
+body {
+	padding: 0;
+	background-image: url(Assets/worldlights.jpg);
+	background-repeat: no-repeat;
+	flex-direction: column;
+	background-attachment: fixed;
+	background-size: cover;
+	background-color: #464646;
+	display: flex;
+	align-items: center;
+	font-size: 1.5rem;
+	color: #feda4a;
+}
+nav {
+	font-size: 1.5rem;
+	width: 70%;
+	margin: 30px auto;
+}
+nav:hover {
+	opacity: 1;
+}
+
+.home {
+	margin-left: 15px;
+	padding: 10px;
+	border-bottom: 2px solid;
+	border-color: transparent;
+	float: right;
+}
+.home:hover {
+	border-bottom: 2px solid;
+	border-color: #2bd128;
+}
+@media only screen and (max-width: 905px) {
+	.home {
+		background-image: url(Assets/home.png);
+		background-size: cover;
+		height: 50px;
+		width: 50px;
+		padding: 0px;
+		float: right;
```

Comment:
This CSS is repeated several times. When you see repeating CSS, it's a good idea to create a generic class for the common properties.